docs/claimantmodel/Book.md

```diff
@@ -207,9 +235,12 @@ If you're getting that feeling that this itself now feels like a Claim, then bin
 Using the Claimant Model to describe promises succinctly shows the inherent problem with them; promises need to be verified.
 Anyone _can_ verify an inclusion promise, but making it so that everyone will verify any promise they have seen is generally not feasible.
 For example, in CT this would require that every browser that has been shown a certificate containing one or more SCTs would verify the inclusion in all logs that had issued SCTs after the MMD had passed.
-Even if this was feasible to implement and users were happy to sacrifice having a lightweight browser for something that performed this additional work (using more disk space, network, battery etc in the meantime), there would then need to be a mechanism for these browsers to be able to report SCTs that did not verify.
+This would place a significant burden on clients (impacting network, storage, and battery) and require a mechanism for reporting failures.
 
 The other option is that SCTs are made discoverable to a party of dedicated verifiers who can report non-fulfilled promises to the arbiter.
-This disoverability problem would require a log for SCTs, which then appears to be precisely the same problem as logging certificates in the first place!
+This discoverability problem would require a log for SCTs, which then appears to be precisely the same problem as logging certificates in the first place!
+
+#### Alternatives to SCTs
+
+The simplest solution is to reduce or eliminate the long merge delay. If a log can offer an MMD of a few seconds, a Believer can simply wait for the inclusion proof to be available before acting on the Claim. This shifts the burden from Believers to the log operators, who must maintain highly available and performant infrastructure, but it dramatically simplifies the client-side logic.
```

Comment:
Since we're talking about the Believer fetching the inclusion proof I think it's worth adding a parenthetical about the alternative "claimant can wait for the inclusion proof to be available and make an off-line verifiable bundle which includes the claim available for direct download"). Particularly if we mention the privacy issues with SCTs in the paras above...

Reply:
Dodged this to avoid a discussion of offline proofs etc here. Changed believer to user. LG?